lib/classes/ConfigSchemaHandler/index.js

```diff
@@ -136,6 +136,15 @@ class ConfigSchemaHandler {
             this.serverless.cli.log(`  ${message}`, 'Serverless', { color: 'orange' });
           }
         }
+        if (this.serverless.configurationInput.configValidationMode === undefined) {
```

Comment:
I believe we should be able to use conditional as `!this.serverless.configurationInput.configValidationMode` or we have to check against `undefined` explicitly?